### DISCUSSION TITLE
updated yesterday, tried network graph and its hanging. what do?

### DISCUSSION THREAD

**Discussion by spikenheimer** [zwave-js/zwave-js-ui#1643] [General]
hey all,

i updated my setup last night to 8.2.1/5.5.2 after loving this for the last few months.  (i've had a journey with my zwave across vera, smartthings, hubitat and now to HASS, but honestly when i started to migrate off hubitat i was tempted to just run zwavejs2 w/ a homebridge alone lol) 
anyways on a whim i tried using the network graph last today and i noticed the webpage hangs - when i tried again after starting a separate debug window, i could see the INFO ZWAVE: Success zwave api call refreshNeighbors completing with the json resultset... my question is what's the best way to debug this or is it just broken for the size of my network? ( 106 nodes plus my controller )

**Comment by robertsLando** [comment]
> or is it just broken for the size of my network

Could you try to see develooper console on your browser to see if there are errors there? Does the window stop responding when opening the mesh page?

**Reply by spikenheimer** [reply]
i tried that but the dev console shows nothing (or i'm doing it wrong) and the window stops responding when i got to /mesh

**Reply by robertsLando** [reply]
Could attach here a nodes dump? you can get it from control panel > advanced > nodes dump

**Reply by spikenheimer** [reply]
whoa. sorry about how long this took, i never got a notification on your reply :/

here attached is the nodes dump.
[nodes_dump.json.txt](https://github.com/zwave-js/zwavejs2mqtt/files/7207598/nodes_dump.json.txt)

**Reply by spikenheimer** [reply]
after updating this weekend to zwavejs2mqtt: 5.7.1.885441e / zwave-js: 8.4.0 and also having the updated Safari 15.0 i noticed that the network map is working again. 🤷‍♂️

**Reply by robertsLando** [reply]
Sorry for no reply, I had some busy weeks. BTW glad it works now :)
